src/zkevm_specs/evm/execution/error_depth.py

```diff
@@ -0,0 +1,39 @@
+from ...util import FQ
+from ..instruction import Instruction, Transition
+from ..opcode import Opcode
+from ..table import CallContextFieldTag
+
+
+def error_depth(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+    # current executing op code must be CALL-family
+    instruction.constrain_in(
+        opcode,
+        [FQ(Opcode.CALL), FQ(Opcode.CALLCODE), FQ(Opcode.DELEGATECALL), FQ(Opcode.STATICCALL)]
+    )
+
+    is_call, is_callcode, is_delegatecall, is_staticcall = instruction.multiple_select(
```

Comment:
don't need to check it is call or call code etc.

Reply:
but CALL and CALLCODE will consume 7 stack elements and DELEGATECALL and STATICCALL will consume 6 elements